testing-2023-05.md

```diff
@@ -0,0 +1,82 @@
+# napari metadata plugin testing
+
+Author: Lucy Obus (@lcobus)
```

Comment:
```suggestion
Author: [Lucy Obus](https://github.com/LCObus)
```

Reply:
This is just markdown, so in general using @ to mention github user names can be confusing (outside the context of GitHub) and also doesn't really seem to work even when the Markdown is rendered on GitHub. Changed the @ mentions to full names with links to GitHub user pages.